macrodata.js

```diff
@@ -0,0 +1,59 @@
+// TODO: add more file names
+const files = [
+  'Siena',
```

Comment:
I think these are all from episode 2, there are more shown in episode 1. Could we crowdsource more?